mods/cnc/rules/campaign-tooltips.yaml

```diff
@@ -55,8 +55,9 @@
 ^CommonHuskDefaults:
 	Tooltip:
 		GenericVisibility: Enemy, Ally, Neutral
-		GenericStancePrefix: false
 		ShowOwnerRow: false
+		AllyPrefix:
```

Comment:
I'm not a big fan of having to override these to empty.  IMO it would be better to keep the `GenericStancePrefix` as well as the new prefix overrides.  What do the other @OpenRA/engine-hackers think?